src/app/qgsrelationadddlg.cpp

```diff
@@ -141,17 +139,17 @@ void QgsRelationAddDlg::updateFieldsMappingHeaders()
   mFieldsMappingTable->setVerticalHeaderLabels( verticalHeaderLabels );
 }
 
-QString QgsRelationAddDlg::referencingLayerId()
+QString QgsRelationAddDlg::referencingLayerId() const
```

Comment:
This makes me think we should harmonize our use of Dlg vs Dialog class name in QGIS 4.0.

In the meantime, might be worth killing the _only_ use of dlg in src/app, that is QgsRelationAddDlg :) doesn't have to happen in this PR.

Reply:
Ok, rename to QgsCreateRelationDialog